mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerLocalScan.java

```diff
@@ -311,6 +313,26 @@ public void testCreateTableWithColumnSpecificationMultilevelPartitioned() throws
     runCreateAndReadTest(identifier, createSql, HiveIcebergStorageHandlerTestUtils.CUSTOMER_SCHEMA, spec, data);
   }
 
+  @Test
+  public void testExternalTableToTableManagedInHiveCatalog() throws IOException {
```

Comment:
If we make these alias tables supported, I think it would make sense to add a few more tests, and some scenarios to think about, e.g.:
- writing into the aliased table (unless it's intended to be read-only, but I think with the current construct it's not)
- dropping the referenced table - the alias table becomes unreadable 
- the supplied property at create table time points to a non-existing table